components/Vote/201912/VotePage.js

```diff
@@ -0,0 +1,460 @@
+                        id,
+                        data: data[slug]
+                      }))}
+                    />
+                    <Body dangerousHTML={vt('vote/201912/result/after')} />
+                    <div style={{ height: 80 }} />
+                  </Fragment>
+                )}
+                {hasEnded && !hasResults && (
+                  <div {...styles.thankyou}>
+                    <RawHtml
+                      type={P}
+                      dangerouslySetInnerHTML={{
+                        __html: vt('vote/201912/ended')
+                      }}
+                    />
+                  </div>
+                )}
+
+                <Title>{vt('vote/201912/title')}</Title>
+                {actionBar}
+                <div {...styles.image}>
+                  <FigureImage
+                    src={`${CDN_FRONTEND_BASE_URL}/static/genossenschaft/urabstimmung3.jpg?resize=780x`}
+                  />
+                  <FigureCaption>{vt('vote/201912/caption')}</FigureCaption>
+                </div>
+                {missingAdress && (
+                  <Fragment>
+                    <a {...styles.anchor} id='adresse' />
+                    <H2>{vt('common/missingAddressTitle')}</H2>
+                    <P>{vt('common/missingAddressBody')}</P>
+                    <div style={{ margin: '30px 0' }}>
+                      <AddressEditor />
+                    </div>
+                  </Fragment>
+                )}
+                {!me && !hasEnded && (
+                  <div style={{ margin: '80px 0' }}>
+                    <SignIn
+                      beforeForm={
+                        <Fragment>
+                          <H2>{vt('common/signInTitle')}</H2>
+                          <RawHtml
+                            type={P}
+                            dangerouslySetInnerHTML={{
+                              __html: vt('vote/201912/signInBody')
+                            }}
+                          />
+                        </Fragment>
+                      }
+                    />
+                  </div>
+                )}
+                {md(mdComponents)`
+Sehr geehrte Verlegerin, sehr geehrter Verleger
+
+Herzlich willkommen zur dritten Urabstimmung von Project R. Viele von Ihnen haben bereits Übung: Im Herbst 2018 haben Sie gewählt und über die Finanzen abgestimmt und im Sommer 2019 die Revisionsstelle bestätigt sowie das Budget verabschiedet.
+
+Dieses Mal geht es um die Genehmigung des  vergangenen Geschäftsjahres. Oder im Fachjargon: Das sind die Traktanden dieser Urabstimmung: 
+
+ - [Geschäftsbericht 2018/2019](#geschaftsbericht)
+ - [Jahresrechnung Project R 2018/2019](#jahresrechnung)
+ - [Entlastung des Vorstandes](#entlastung)
+ - [Wahl der Revisionsstelle 2019/2020](#revisionsstelle)
+
+Die virtuellen Urnen sind vom 13. bis zum 23. Dezember 2019 geöffnet. **${numVotes} Ihrer Kolleginnen und Kollegen haben bereits abgestimmt.**
+
+${(
```

Comment:
you propbably get invalid dom nesting with this because ouf `markdown-in-js` generating a p tag around `Collapsible` and the p tags agin inside. Solution:
```
`}
<Collapsible>{md`
...
`}</Collapsible>
{md(mdComponents)`
```